modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss

```diff
@@ -3,6 +3,15 @@
 	.ddm-translation-manager {
 		margin-top: 1.25rem;
 
+		.lfr-translationmanager {
+			width: 70%;
+			z-index: 0;
+
+			@media (max-width: 1180px) {
+				width: 50%;
```

Comment:
Since we work with Bootstrap I suggest you to change the max-width into `992px` that is the equivalent of the max size of medium screens (you can see all the sizes [here](https://bootstrapcreative.com/bootstrap-4-media-queries/)), keeping the pattern that is already in use.

There are cases that is needed to use custom media-queries, but while testing your solution I realized that using `992px` would be better because of the spacing. I've attached a video to show you the result, what do you think?

https://user-images.githubusercontent.com/60899738/186016126-0721dad1-83d2-4c17-a8b8-badaf5bac63a.mp4

Reply:
If there is an specific reason to use `1180px` could you explain me then? Thanks a lot!

Reply:
Hi @rebsilva, no specific reason, just testing I arrived at 1180px, but I will make the switch to 992px as suggested, thanks!